test/fromPairs.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var S = require('..');
+
+var eq = require('./internal/eq');
+
+
+test('fromPairs', function() {
+
+  eq(typeof S.fromPairs, 'function');
+  eq(S.fromPairs.length, 1);
+  eq(S.fromPairs.toString(), 'fromPairs :: Array (Pair String a) -> StrMap a');
+
+  eq(S.fromPairs([]), {});
+  eq(S.fromPairs([['a', 1], ['b', 2], ['c', 3]]), {a: 1, b: 2, c: 3});
+  eq(S.fromPairs([['x', 1], ['x', 2]]), {x: 2});
```

Comment:
I just realized that this would also make an excellent doctest. It neatly demonstrates the second sentence of the description. Would you mind adding this to the `` ```javascript `` block as well?